package.json

```diff
@@ -0,0 +1,45 @@
+{
+  "name": "react-components",
+  "private": true,
+  "version": "0.0.0",
+  "type": "module",
+  "scripts": {
+    "dev": "vite",
+    "build": "tsc && vite build",
+    "lint": "eslint . --ext ts,tsx --report-unused-disable-directives --max-warnings 0",
+    "preview": "vite preview",
+    "format:fix": "prettier . --write",
+    "prepare": "husky install",
+    "pre-commit": "eslint . --ext ts,tsx --report-unused-disable-directives --max-warnings 0 && git add ."
+  },
+  "husky": {
+    "hooks": {
+      "pre-commit": "prettier --write . && git add ."
```

Comment:
This code doesn't take part in husky hooks since you defined it in the .husky folder. So making a commit it runs a command from 13th line, not this one